src/main/java/com/wego/dao/impl/guava/GuavaCachedResponseImpl.java

```diff
@@ -0,0 +1,55 @@
+package com.wego.dao.impl.guava;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+import com.wego.dao.CachedResponseDao;
+import com.wego.dao.entities.CachedResponseEntity;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+public class GuavaCachedResponseImpl implements CachedResponseDao {
+  private static long CACHE_TTL = 30;
+  private static Cache<String, String> CACHE =
+      CacheBuilder.newBuilder().expireAfterWrite(CACHE_TTL, TimeUnit.MINUTES).build();
+
+  public static final ObjectMapper OBJECT_MAPPER =
+      new ObjectMapper()
+          .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+          .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
+
+  @Override
+  public CachedResponseEntity save(CachedResponseEntity cachedResponseEntity) {
+    String jsonEntity = null;
+    try {
+      jsonEntity = OBJECT_MAPPER.writeValueAsString(cachedResponseEntity);
+    } catch (JsonProcessingException ex) {
+      ex.printStackTrace();
```

Comment:
definitely, I will add it later, btw GuavaCachedResponseImpl is just an example how to implement CachedResponse, we are not going to use this, We will implement `RedisCachedResponseImpl` in our curiosity/akasha. This is the only one thing you need to do in order to use this lib.